nck/readers/dbm_reader.py

```diff
@@ -230,7 +233,7 @@ def read(self):
         elif request_type == "lineitems_objects":
             data = self.get_lineitems_objects()
         else:
-            raise Exception("Unknown request type")
+            raise ClickException("Unknown request type")
```

Comment:
I think this is useless to check that here anyway because a correct request type is forced by Click thanks to the `Click.Choice(POSSIBLE_REQUEST_TYPES)`.